src/NetherPortalScanner.cpp

```diff
@@ -296,8 +297,18 @@ void cNetherPortalScanner::OnDisabled(void)
 		Position.z += OutOffset;
 	}
 
-	FLOGD("Placing player at {0}", Position);
-	m_Entity->MoveToWorld(m_World, Position, true, false);
+	// Lookup our warping entity by ID
+	// Necessary as they may have been destroyed in the meantime (#4582)
+	m_SourceWorld.DoWithEntityByID(
+		m_EntityID,
+		[this, &Position](auto & a_Entity)
```

Comment:
That's a c++14 feature
```suggestion
		[this, &Position](cEntity & a_Entity)
```